### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 1 - Esercizio 8

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#474] [Esercizi M-Z]
![foglio 1 es.8](https://github.com/user-attachments/assets/85fcb9d5-fa1d-4d5b-9d5a-9cc6925d7851)

**Comment by matypist** [comment]
Soluzioni per le lettere $c$, $d$, $f$, $g$, ad opera della [tutor](https://github.com/sapienzastudentsnetwork/algebra/blob/main/scandiuzzi/README.md#aa-202425-scandiuzzi-tutor) Elisa **Scandiuzzi**, lezione del **8 ottobre** 2024 (`appunti_24_10_08.pdf`):

![image](https://github.com/user-attachments/assets/cc25a974-b878-4529-a1b9-3c3b1a0cd467)

![image](https://github.com/user-attachments/assets/12d9660d-fec9-4ad1-ab3c-50b2ccad65cb)
